content/bn/docs/reference/glossary/container-runtime.md

```diff
@@ -0,0 +1,22 @@
+---
+title: কন্টেইনার রানটাইম
+id: container-runtime
+date: 2019-06-05
+full_link: /bn/docs/setup/production-environment/container-runtimes
```

Comment:
```suggestion
full_link: /docs/setup/production-environment/container-runtimes/
```

because that page is not yet localized.

Reply:
Ok. 
So after The page is localized we can change it back to `full_link: /bn/docs/setup/production-environment/container-runtimes`?

Reply:
Yes.